MM/6.2.0/config/media_manager_5/fields/main-menu-custom-link-icon.dcl

```diff
@@ -0,0 +1,9 @@
+resource configservice_string_config_field main_menu_custom_link_icon {
+     type = 'String'
+     product_id = resource.configservice_product.media_manager_5.id
+     default_value = 'icon-help'
+     group = 'Main menu'
+     key = 'customLinkIcon'
+     title = 'Custom link icon'
+     description = 'Possible values: icon-help, icon-manual, icon-info'
+}
```

Comment:
I think that a `configservice_combo_config_field` would be more appropriate here since it seems like we only support three very specific values?